internal/pkg/ccloudv2/schema-registry.go

```diff
@@ -0,0 +1,30 @@
+package ccloudv2
+
+import (
+	"context"
+
+	srcm "github.com/confluentinc/ccloud-sdk-go-v2/srcm/v2"
+)
+
+func newSchemaRegistryClient(url, userAgent string, unsafeTrace bool) *srcm.APIClient {
+	cfg := srcm.NewConfiguration()
+	cfg.Debug = unsafeTrace
+	cfg.HTTPClient = newRetryableHttpClient(unsafeTrace)
+	cfg.Servers = srcm.ServerConfigurations{{URL: url}}
+	cfg.UserAgent = userAgent
+
+	return srcm.NewAPIClient(cfg)
+}
+
+func (c *Client) SchemaRegistryApiContext() context.Context {
+	return context.WithValue(context.Background(), srcm.ContextAccessToken, c.AuthToken)
+}
+
```

Comment:
Can you match the style of the other files in this package, and include all of your helper functions here?

Reply:
They may be used multiple times throughout the code, so we one want to write them once.